packages/volto/src/actions/favorites/favorites.js

```diff
@@ -0,0 +1,36 @@
+import { cloneBlocks } from '@plone/volto/helpers/Blocks/cloneBlocks';
+import config from '@plone/volto/registry';
+
+export const ADD_TO_FAVORITES = 'ADD_TO_FAVORITES';
+export const REMOVE_FROM_FAVORITES = 'REMOVE_FROM_FAVORITES';
+export const LOAD_FAVORITES = 'LOAD_FAVORITES';
+
+export function loadFavorites() {
+  return {
+    type: LOAD_FAVORITES,
+  };
+}
+
+export function addToFavorites(blockData) {
+  const { id, type, data } = blockData;
```

Comment:
when you call this actions, isn'it already cloned data from here https://github.com/plone/volto/pull/6553/files#diff-d4b3b0bf1bb3bfa0fb3a87a795eeac0acb302fca3c1e1bf60be5ce9cf516f894R44?
why you need to clone again?